README.md

```diff
@@ -2,138 +2,16 @@
 ![Python](https://img.shields.io/badge/built%20with-Python3-red.svg)
 ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)
 
-[español](https://github.com/GramAddict/bot/blob/master/res/README_es.md) | [português](https://github.com/GramAddict/bot/blob/master/res/README_pt_BR.md)
-
-Liking and following automatically on your Android phone/tablet. No root required: it works on [uiautomator2](https://github.com/openatx/uiautomator2), which is a faster and more efficient fork of the official Android UI testing framework [UI Automator](https://developer.android.com/training/testing/ui-automator). This is a completely free and open source project that is forked from the freemium project [Insomniac](https://github.com/alexal1/Insomniac/). You can check them out, but you'll probably like us better. 😊
-
 <img src="https://github.com/GramAddict/bot/raw/master/res/demo.gif">
```

Comment:
It needs to be "prettied" up some more, but I'm waiting on a couple things to do that. I edited it so that it has basic info in again. More work to be done soon.